Bottom: dbb24b900018a9627ea7ad901c186abcabb5229a
Top:    18854afb6b2fef4d972541319707eef7cabe0229
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-28 10:46:37 -0700

Refresh of rstream-opt

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 067b746..6577fe1 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -379,12 +379,12 @@ static void set_options(int rs)
 
 	if (use_rs) {
 		/* Inline size based on experimental data */
-//		if (optimization == opt_latency)
-//			val = 384;
-//		else if (optimization == opt_bandwidth)
-			val = 32; // 0;
-//		else
-//			val = 64;
+		if (optimization == opt_latency)
+			val = 384;
+		else if (optimization == opt_bandwidth)
+			val = 0;
+		else
+			val = 64;
 
 		rs_setsockopt(rs, SOL_RDMA, RDMA_INLINE, &val, sizeof val);
 	}
